src/cgls/cgls.c

```diff
@@ -40,7 +40,13 @@
 static bool arg_no_pager = false;
 static bool arg_kernel_threads = false;
 static bool arg_all = false;
-static bool arg_unit = false;
+
+static enum {
+        SHOW_UNIT_NONE,
+        SHOW_UNIT,
```

Comment:
to make this less confusing I'd probably call this internal enum SHOW_UNIT_SYSTEM, even if the exposed parameter is --unit= and not --system-unit...